training/finetune_acft_hf_nlp/environments/acpt/context/Dockerfile

```diff
@@ -6,10 +6,11 @@ RUN ls -lh
 
 USER root
 RUN apt-get -y update
+RUN pip install --upgrade pip
 
 COPY requirements.txt .
 
-RUN pip install "azureml-evaluate-mlflow=={{latest-pypi-version}}"
+RUN pip install "azureml-evaluate-mlflow==0.0.6"
```

Comment:
What is the need to pin a specific version?

Reply:
@skasturi I am trying with different versions; the environment which worked yesterday is failing due to conflicts. I am checking which is causing the issue.

Reply:
reverted to latest version